.gitignore

```diff
@@ -2,3 +2,5 @@
 *.mem
 Manifest.toml
 LocalPreferences.toml
+
+.vscode/
```

Comment:
This should go in your own excludes file, see https://git-scm.com/docs/gitignore